crawl-ref/source/skills.cc

```diff
@@ -1688,13 +1688,14 @@ string skill_title_by_rank(skill_type best_skill, uint8_t skill_rank,
 
         case SK_UNARMED_COMBAT:
             if (species == SP_FELID)
-            {
                 result = claw_and_tooth_titles[skill_rank];
-                break;
+            else if (species == SP_MUMMY && skill_rank == 5 && god == GOD_NO_GOD)
```

Comment:
It breaks with the meme but remove the god check and I'll merge this.

Reply:
Yeah, that's fair.

(Despite the fact that I want to grumble about it~)

Reply:
Okay, I've removed the god check and rebased to master.